spec/spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 require "simplecov" if ENV["COVERAGE"] == "true"
 
+Dir[File.expand_path("#{File.dirname(__FILE__)}/support/matchers/*.rb")].each { |file| require file }
```

Comment:
My only piece of feedback/nitpick would be that `require_relative "support/matchers/perform_database_query_matcher"` feel simpler and more explicit to me. I also liked the fact that this was only required lazily when needed and I think it might've been helpful to understand that this gem was only needed for two specs.